packages/ag-charts-types/src/chart/axisOptions.ts

```diff
@@ -124,7 +124,7 @@ export interface AgBaseAxisLabelStyleOptions {
     /** The font family to use for the labels */
     fontFamily?: FontFamily;
     /** Padding in pixels between the axis label and the tick. */
-    padding?: PixelSize;
+    spacing?: PixelSize;
```

Comment:
Please document breaking changes in the `Breaking Changes` field of the related JIRA, here and below.

Reply:
Added a separate ticket to track this change:
https://ag-grid.atlassian.net/browse/AG-13509